src/main/java/com/regnosys/testing/pipeline/PipelineTreeConfig.java

```diff
@@ -45,6 +45,11 @@ public class PipelineTreeConfig {
     private boolean strictUniqueIds;
     private Path writePath;
     private Predicate<String> testPackIdFilter = testPackId -> true;
+    private String prefix;
+
```

Comment:
You could also add a zero arg constructor, i.e.
```
    public PipelineTreeConfig() {
        this(null);
    }
```
So you don't have to do this `new PipelineTreeConfig(null)`